pkg/kubelet/apis/cri/v1alpha1/api.proto

```diff
@@ -971,33 +977,75 @@ message UInt64Value {
     uint64 value = 1;
 }
 
-// FsInfo contains data about filesystem usage.
-message FsInfo {
-    // The block device name associated with the filesystem.
-    string device = 1;
-    // The root directory for the images.
-    string path = 2;
-    // CapacityBytes represents the total capacity (bytes) of the filesystems
-    // underlying storage.
-    UInt64Value capacity_bytes = 3;
-    // AvailableBytes represents the storage space available (bytes) for the
-    // filesystem.
-    UInt64Value available_bytes = 4;
+// StorageIdentifier uniquely identify the storage..
+message StorageIdentifier{
+    // UUID of the device.
+    string uuid = 1;
```

Comment:
UUID may be same in some cases, e.g. multiple disks from same ceph volume (e.g. original volume and snapshot) could have same UUID. Replace uuid with scsi id?

cc/ @vishh @Random-Liu WDYT?